CHANGELOG.md

```diff
@@ -1,3 +1,49 @@
+## Fleet 4.20.0 (Sep 8, 2022)
+
+* Improved vulnerability detection for macOS hosts by improving detection of Zoom, Ruby, and Node.js vulnerabilities. Warning: For users that download and sync Fleet's vulnerability feeds manually, there are [required adjustments](https://github.com/fleetdm/fleet/issues/6628) or else vulnerability processing will stop working. Users with the default vulnerability processing settings can safely upgrade without adjustments.
```

Comment:
@michalnicp do you have any concerns with the "Improved vulnerability detection" entry in the CHANGELOG? I'm linking to the GitHub issue, that includes the in depth instructions, to keep the entry brief.

Reply:
That looks good to me.